linera-execution/src/execution.rs

```diff
@@ -21,13 +21,13 @@ use linera_views::{
     views::{View, ViewError},
 };
 use linera_views_derive::CryptoHashView;
+use std::collections::BTreeMap;
 
 #[cfg(any(test, feature = "test"))]
 use {
     crate::{system::SystemExecutionState, TestExecutionRuntimeContext, UserApplicationCode},
     async_lock::Mutex,
     linera_views::memory::{MemoryContext, TEST_MEMORY_MAX_STREAM_QUERIES},
-    std::collections::BTreeMap,
     std::sync::Arc,
 };
 
```

Comment:
Ok so personally I was expecting storage sizes to be handled in `KeyValueStoreView` so I'm a bit surprised by the current approach. Let's discuss more about this offline.